examples/rag_on_wikipedia_article/retrieve_documents.py

```diff
@@ -0,0 +1,32 @@
+"""Retrieve documents from the Chroma vector store."""
+import os
+
+from rag_config import settings
+from stores.wikipedia_chroma import WikipediaStore
+
+from mirascope.anthropic import AnthropicCall
+
+if settings.anthropic_api_key:
+    os.environ["ANTHROPIC_API_KEY"] = settings.anthropic_api_key
+if settings.openai_api_key:
+    os.environ["OPENAI_API_KEY"] = settings.openai_api_key
+
+
+class WikipediaCall(AnthropicCall):
+    prompt_template = """
+    SYSTEM: Answer the question based on the context.
+    {context}
+    USER: {question}
+    """
+
+    question: str
+
+    @property
+    def context(self):
+        store = WikipediaStore()
```

Comment:
is there any concern with creating the store instance every call?

For example, in a call with history where I'll update an attribute and make another call this will create a new store instance each call instead of just creating the instance on class instantiation.

Reply:
it will just retrieve the chroma collection instance.